lua/bufferline.lua

```diff
@@ -81,12 +81,14 @@ function bufferline.setup(user_config)
 
   create_user_command(
     'BufferMoveStart',
-    function(tbl) api.move_current_buffer_to(1) end,
```

Comment:
Unused `tbl` param; removed to clean the workspace diagnostics. Otherwise unrelated.